pyvcloud/vcd/org.py

```diff
@@ -638,3 +642,134 @@ def update_org(self, org_name, is_enabled=None):
                                                 org_admin_resource,
                                                 EntityType.ADMIN_ORG.value)
         return org_admin_resource
+
+    def create_org_vdc(self,
+                       vdc_name,
+                       provider_vdc_name,
+                       description='',
+                       allocation_model='AllocationVApp',
+                       cpu_units='MHz',
+                       cpu_allocated=0,
+                       cpu_limit=0,
+                       mem_units='MB',
+                       mem_allocated=0,
+                       mem_limit=0,
+                       nic_quota=0,
+                       network_quota=0,
+                       vm_quota=0,
+                       storage_profiles=[],
+                       resource_guaranteed_memory=None,
+                       resource_guaranteed_cpu=None,
+                       vcpu_in_mhz=None,
+                       is_thin_provision=None,
+                       network_pool_name=None,
+                       uses_fast_provisioning=None,
+                       over_commit_allowed=None,
+                       vm_discovery_enabled=None,
+                       is_enabled=True):
+        """
+        Create Organization VDC in the current Org.
+        :param vdc_name (str): The name of the new org vdc.
+        :param provider_vdc_name (str): The name of the new provider vdc.
```

Comment:
You mean name of the 'existing' provider vdc?

Reply:
yes, will update.